schema.json

```diff
@@ -4055,8 +4055,8 @@
             },
             {
               "defaultValue": null,
-              "description": "",
-              "name": "apartmentNumber",
+              "description": "A New York City borough.",
+              "name": "borough",
```

Comment:
Urg, all the changes in `schema.json` are due to the fact that our use of `ModelForm` is re-ordering the fields; this doesn't actually make any practical difference, it just changes the order in which the fields are listed in our `schema.json`.